pkg/artifact/spec.md

```diff
@@ -13,7 +13,7 @@ The artifacts and their media types are
 
 **ModelKit File (Kitfile)** Acts as a record detailing the properties, relationships, and intended uses of the included artifacts. The Kitfile is central to understanding the structure and purpose of a ModelKit. It adopts the `application/vnd.kitops.modelkit.config.v1+json` media type for easy access and interpretation by tools.See the seperate kitfile specification on details
 
-**ModelKit Manifest:** This JSON document provides essential information about the model, including creation date, authorship, and a cryptographic hash of each artifact and the Kitfile. The manifest is immutable to preserve the integrity of the ModelKitID, ensuring any modification results in the creation of a new derived ModelKit, rather than altering the existing one.
+**ModelKit Manifest:** This JSON document provides essential information about the model, including creation date, authorship, and a cryptographic hash of each artifact and the Kitfile. The manifest is immutable to preserve the integrity of the ModelKitID, ensuring any modification results in the creation of a new derived ModelKit, rather than altering the existing one. ModelKit manifest adheres to the [OCI Image Manifest Specification](https://github.com/opencontainers/image-spec/blob/6a983fd8be10f63063ce6452be099cd6e20fb36b/manifest.md) ensuring compatibility with OCI artifact registries.
```

Comment:
```suggestion
**ModelKit Manifest:** This JSON document provides essential information about the model, including creation date, authorship, and a cryptographic hash of each artifact and the Kitfile. The manifest is immutable to preserve the integrity of the ModelKitID, ensuring any modification results in the creation of a new derived ModelKit, rather than altering the existing one. ModelKit manifests adhere to the [OCI Image Manifest Specification](https://github.com/opencontainers/image-spec/blob/6a983fd8be10f63063ce6452be099cd6e20fb36b/manifest.md) ensuring compatibility with OCI artifact registries.
```